src/ray/raylet/node_manager.cc

```diff
@@ -351,14 +365,45 @@ void NodeManager::HeartbeatAdded(gcs::AsyncGcsClient *client, const ClientID &cl
                   << client_id;
     return;
   }
-  SchedulingResources &resources = it->second;
-  ResourceSet heartbeat_resource_available(heartbeat_data.resources_available_label,
-                                           heartbeat_data.resources_available_capacity);
-  resources.SetAvailableResources(
+  SchedulingResources &remote_resources = it->second;
+
+  ResourceSet remote_available(heartbeat_data.resources_available_label,
+                               heartbeat_data.resources_available_capacity);
+  ResourceSet remote_load(heartbeat_data.resource_load_label,
+                                      heartbeat_data.resource_load_capacity);
+  // TODO(atumanov): assert that the load is a non-empty ResourceSet.
+  RAY_LOG(DEBUG) << "[HeartbeatAdded]: received load: "
+                << remote_load.ToString();
+  remote_resources.SetAvailableResources(
       ResourceSet(heartbeat_data.resources_available_label,
```

Comment:
Can you replace the argument here with `remote_available`? They look the same to me. It also seems like you could keep this and remove `remote_available` and `remote_load` entirely.